prisma/send-notifications.ts

```diff
@@ -0,0 +1,187 @@
+import { firebaseService, prisma } from '../src/firebase.js';
```

Comment:
I think a better name for this file would be `notificationService.ts` instead of the current - it makes more sense intuitively as this file is meant to run on a different container from the main app